_posts/2017/2017-09-06-napalm-logs-released.md

```diff
@@ -0,0 +1,325 @@
+---
+title: "Using napalm-logs for event-driven network automation and orchestration"
+author: mirceaulinic
+description: "Using napalm-logs for event-driven network automation and orchestration"
+tags: ["post", "napalm-logs", "yang"]
+---
+
+After months of continuous work, we are thrilled to announce the first major
+release of [*napalm-logs*](https://github.com/napalm-automation/napalm-logs).
+
+# What is napalm-logs
+
+*napalm-logs* is open source software acting as a daemon, which listens for
+syslog messages from network devices and normalizes them into a vendor-agnostic
+shape. Unlike the rest of the libraries maintained by the NAPALM automation
+community, *napalm-logs* does not connect to the device to retrieve information,
+but rather receives messages.
+
```

Comment:
Can you perhaps insert a `<!--more-->` tag somewhere around here? Since the post is quite long this would case a break when viewing the /blog/ page.

Reply:
You mean like this? https://github.com/napalm-automation/napalm-automation.github.io/commit/da4c00f

Reply:
👍